src/components/LineChart.jsx

```diff
@@ -13,12 +13,11 @@ const calculateHeight = (total) => {
 };
 
 const calculateMinMax = (data) => {
-    const minValue = Math.min(...data);
     const maxValue = Math.max(...data);
     // calculate a dynamic value to center the graph
```

Comment:
We're no longer _centering_ things here, so I'm not sure how well this comment applies anymore.